lua/xcodebuild/core/xcode.lua

```diff
@@ -512,18 +512,28 @@ end
 ---Installs the application on simulator.
 ---@param destination string
 ---@param appPath string
+---@param retrying boolean|nil
 ---@param callback function|nil
 ---@return number # job id
-function M.install_app_on_simulator(destination, appPath, callback)
+function M.install_app_on_simulator(destination, appPath, retrying, callback)
   local command = "xcrun simctl install '" .. destination .. "' '" .. appPath .. "'"
 
   return vim.fn.jobstart(command, {
     stdout_buffered = true,
     on_exit = function(_, code, _)
       if code ~= 0 then
-        notifications.send_error("Could not install app (code: " .. code .. ")")
         if code == 149 then
-          notifications.send_warning("Make sure that the simulator is booted")
+          if retrying then
+            util.call(callback)
+          else
+            local retry = function()
+              M.install_app_on_simulator(destination, appPath, retrying, callback)
+            end
```

Comment:
It looks like, it should be `true` instead of `retrying` here. Otherwise, this parameter will never be `true`